### DISCUSSION TITLE
issue: ReactNative password field not obscuring text

### DISCUSSION THREAD

**Discussion by odeke** [react-hook-form/react-hook-form#11857] [Q&A]
### Version Number

7.51.3

### Codesandbox/Expo snack

https://codesandbox.io/p/devbox/reacthookform-n7tvqs?file=%2FApp.js

### Steps to reproduce

1. Start a new react native expo project, 
2. Install react-hook-form
3. Setup a controller around a  textinput that is a password type, using secureTextEntry 
4. Run the app and type in a password.

### Expected behaviour

It should mask the password content 

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

**Comment by longb1997** [comment]
any updated on this?
